pkg/ring/kv/etcd/etcd.go

```diff
@@ -41,6 +42,7 @@ func New(cfg Config, codec codec.Codec) (*Client, error) {
 	cli, err := clientv3.New(clientv3.Config{
 		Endpoints:   cfg.Endpoints,
 		DialTimeout: cfg.DialTimeout,
+		DialOptions: []grpc.DialOption{grpc.WithBlock()},
```

Comment:
I'm not sure adding this is correct. The etcd client we have in master already has the new client balancer merged into (https://github.com/etcd-io/etcd/pull/9860). To preserve the current logic we should **not** add this. Have you found out anything different?

Reply:
My bad, you're right, this change was made in June 2018, I mixed it up.